samples/05_content_publishers/publishing_sd_shapefiles_and_csv.ipynb

```diff
@@ -65,7 +65,7 @@
     "            search_result = gis.content.search(query=file_name, item_type=current_item_type)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        item = root_folder.add({}, data_dir + current_sd_file)   # .sd file is uploaded and a .sd file item is created
</pre><p>Need "title" and "item_type" for item_properties param, and ".result()" to get the Folder.add result - Same in some of the Folder.add code below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>